### DISCUSSION TITLE
how bfe handle old and new quic connections when restart.

### DISCUSSION THREAD

**Discussion by yfming** [bfenetworks/bfe#1065] [General]
how bfe handle old and new quic connections when restart.
when restart, if bfe can keep quic/http3 connections open to complete old connections.
and if new quic/http3 connections can not still be handled by old workers.

**Comment by iyangsj** [comment]
@yfming The BFE open source version does not currently support QUIC. If you are referring to the BFE private version, please consult the relevant person privately.
